code/game/turfs/simulated/floor.dm

```diff
@@ -182,4 +182,13 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 						dirtoverlay = new/obj/effect/decal/cleanable/dirt(src)
 						dirtoverlay.alpha = 10
 					else if(dirt > 100)
-						dirtoverlay.alpha = min(dirtoverlay.alpha+10, 200)
\ No newline at end of file
+						dirtoverlay.alpha = min(dirtoverlay.alpha+10, 200)
+
+/turf/simulated/floor/MouseDrop_T(atom/dropping, mob/user)
+	if(istype(dropping, /obj/item) && istype(dropping.loc, /turf))
+		var/obj/item/I = dropping
+		if(isliving(user) && !user.stat)
+			for(I in I.loc)

```

Comment:
not entirely certain but this smells like you could pull items OUT of boxes/containers erroneously.